src/MerchantCenter/AccountService.php

```diff
@@ -469,6 +473,26 @@ private function maybe_add_merchant_center_url( int $merchant_id ) {
 		}
 	}
 
+	/**
+	 * Get the callback function for linking an Ads account.
+	 *
+	 * @throws Exception When the merchant account hasn't been set yet.
+	 */
+	private function link_ads_account() {
+		if ( ! $this->options->get_merchant_id() ) {
+			throw new Exception( 'A Merchant Center account must be connected' );
+		}
+
+		if ( ! $this->options->get_ads_id() ) {
+			// Return early if the ads account hasn't been created yet.
+			return;
+		}
+
+		// Create link for Merchant and accept it in Ads.
+		$this->container->get( Merchant::class )->link_ads_id( $this->options->get_ads_id() );
+		$this->container->get( Ads::class )->accept_merchant_link( $this->options->get_merchant_id() );
```

Comment:
For easy reference we add any class that's fetched from the container in the comment at the top of the class. This helps us keep track of the list of dependencies (which aren't explicitly passed in the constructor). Can we add the `Ads` class to that list?

```php
 * Container used to access:
 * - CleanupSyncedProducts
 * - Merchant
 * - MerchantAccountState
 * - MerchantCenterService
 * - MerchantIssueTable
 * - MerchantStatuses
 * - Middleware
 * - SiteVerification
 * - ShippingRateTable
 * - ShippingTimeTable
```

Reply:
Added the `Ads` class in docblock.